sources.pkr.hcl

```diff
@@ -1,3 +1,47 @@
+# This source defines all the common settings for any AWS AMI (whatever Operating System)
+source "amazon-ebs" "base" {
+  # profile       = "terraform-developer"
+
+  # AWS API connection
+  access_key      = var.aws_access_key_id
+  secret_key      = var.aws_secret_access_key
+
```

Comment:
```suggestion
```

as discussed together, we have to use https://developer.hashicorp.com/packer/integrations/hashicorp/amazon#environment-variables